Library/ViewModels/RewardPledgeViewModel.swift

```diff
@@ -345,14 +354,20 @@ RewardPledgeViewModelOutputs {
       self.expandDescriptionTappedProperty.signal.mapConst(true)
     )
 
-    self.itemsContainerHidden = self.readMoreContainerViewHidden.map(negate)
+    self.itemsContainerHidden = Signal.combineLatest(
+      reward, self.readMoreContainerViewHidden
+      ).map { reward, readMoreIsHidden in
+        reward.rewardsItems.isEmpty || (!reward.rewardsItems.isEmpty && !readMoreIsHidden)
+      }.skipRepeats()
```

Comment:
this addition is ensuring that the items stack view doesn't cause extra space when it is never supposed to be shown.